android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,6 +2,6 @@
 #   $ ./gradlew wrapper --distribution-type=all --gradle-version=NEW_VERSION
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```

Comment:
Hmm, comparing the console output shown by the scan tool for [v5.6.4](https://scans.gradle.com/s/2szkm2hvdrejy/console-log) and [v6.0.1](https://scans.gradle.com/s/l7q26kntclub6/console-log), there aren't additional warnings.

> an appropriate Gradle command

Aha, so I just tried this with `tools/test android --full` on v5.5.1, v5.6.4, and v6.0.1. In the output, between the two colorful "Installing unimodules" sections (which we wish didn't show up, as you've said before), there are some notices.

At all three of those versions, those notices are, modulo some changes in ordering (probably because the build process uses parallelization) highlighted by an online diff checker:

```
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/expo-constants/android/src/main/java/expo/modules/constants/ConstantsService.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/expo-application/android/src/main/java/expo/modules/application/ApplicationModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/@react-native-community/async-storage/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/@react-native-community/cameraroll/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/react-native-sound/android/src/main/java/com/zmxv/RNSound/RNSoundModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/@sentry/react-native/android/src/main/java/io/sentry/RNSentryModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/chrisbobbe/dev/zulip-mobile/node_modules/unimodules-app-loader/android/src/main/java/org/unimodules/apploader/AppLoaderProvider.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

Reply:
> At all three of those versions, those notices are

Interesting. Those appear to be entirely disjoint from the ones seen in that build scan! Specifically in the [build scan](https://scans.gradle.com/s/l7q26kntclub6/deprecations) I see:
> BuildListener#buildStarted(Gradle) has been deprecated.	
> This is scheduled to be removed in Gradle 7.0.	
> 32 usages	
> 
> The maven plugin has been deprecated.	
> This is scheduled to be removed in Gradle 7.0.	
> Please use the maven-publish plugin instead.
> 19 usages

Whereas all the warnings in the output you quoted refer to Java source files -- so they seem to be about APIs used within the app, not used by its build system.